Makefile

```diff
@@ -1,38 +1,42 @@
 .DEFAULT_GOAL := help
 .PHONY: help build test
 
+PYPI_TOKEN := $(shell echo ${PYPI_TOKEN})
+PYPI_TEST_TOKEN := $(shell echo ${PYPI_TEST_TOKEN})
+
+
 build: ## Build an application
-	@pipenv run python -m build --no-isolation
+	@poetry build
+
+configure_pypi_publishing: ## Configure publishing to PyPI
```

Comment:
I don't see any usage of this command in other targets, where/when are we supposed to run it?

Reply:
Ah, you know I was thinking that you would publish with poetry commands but I think this isn't needed if `gh-action-pypi-publish` handles it

Reply:
Oh that's right, you had a `publish-test` command so I assume you publish that manually?

You could just set up pypi publishing manually without `configure_pypi_publishing` but if its in Make, its self documenting

Reply:
okay, I see, then we can keep it here for test publishing. I used to run it manually for test-pypi only and the actual one is handled by github action